src/app/space/app-launcher/services/app-launcher-dependency-editor.service.ts

```diff
@@ -0,0 +1,69 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { Headers, Http, RequestOptions, Response } from '@angular/http';
+
+import {
+    DependencyEditorService,
+    HelperService,
+    TokenProvider
+} from 'ngx-forge';
+
+@Injectable()
+export class AppLauncherDependencyEditorService implements DependencyEditorService {
+
+  private END_POINT: string = '';
+  private API_BASE: string = 'booster-catalog/';
+  private ORIGIN: string = '';
+
+  constructor(
+    private http: Http,
+    private helperService: HelperService,
+    private tokenProvider: TokenProvider
+  ) {
+    if (this.helperService) {
+      this.END_POINT = this.helperService.getBackendUrl();
+      this.ORIGIN = this.helperService.getOrigin();
+    }
+  }
+
+  private get options(): Observable<RequestOptions> {
+    let headers = new Headers();
+    headers.append('X-App', this.ORIGIN);
+    return Observable.fromPromise(this.tokenProvider.token.then((token) => {
+      headers.append('Authorization', 'Bearer ' + token);
+      return new RequestOptions({
+          headers: headers
+      });
+    }));
+  }
+
+  getBoosterInfo(missionId: string, runtimeId: string, runtimeVersion: string): Observable<any> {
+    if (missionId && runtimeId) {
+      debugger;
+      let an_rec_url = process.env['ANALYTICS_RECOMMENDER_URL'] || '';
```

Comment:
Please call options and fetch the urls. It is better not to use process directly.